browser/ui/views/tabs/brave_new_tab_button.cc

```diff
@@ -54,34 +70,134 @@ SkPath BraveNewTabButton::GetBorderPath(const gfx::Point& origin,
                        GetContentsBounds().size());
 }
 
+BraveNewTabButton::BraveNewTabButton(TabStrip* tab_strip,
+                                     PressedCallback callback)
+    : NewTabButton(tab_strip, std::move(callback)) {
+  if (!base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs))
+    return;
+
+  text_ = AddChildView(
+      std::make_unique<views::Label>(l10n_util::GetStringUTF16(IDS_NEW_TAB)));
```

Comment:
I don't think we can use `IDS_NEW_TAB` since it's meant for menu's and contains the `&` for the keyboard shortcut. We either need to make a new string or maybe we can just use `IDS_ACCNAME_NEWTAB` since it is the correct string value.